units/spawnhost_start.go

```diff
@@ -59,11 +59,12 @@ func (j *spawnhostStartJob) Run(ctx context.Context) {
 
 	startCloudHost := func(mgr cloud.Manager, h *host.Host, user string) error {
 		if err := mgr.StartInstance(ctx, h, user); err != nil {
-			event.LogHostStartFinished(h.Id, false)
-			return errors.Wrapf(err, "starting spawn host '%s'", h.Id)
+			err = errors.Wrap(err, "starting spawn host")
+			event.LogHostStartError(h.Id, err.Error())
```

Comment:
Do we need the wrapped error since it'll always start with the same thing, and the event log itself has the started event?

Reply:
Probably not. I moved the wrap back to the return value.